src/components/Tag/Tabs/VulnerabilitiesDetails.jsx

```diff
@@ -124,11 +168,38 @@ function VulnerabilitiesDetails(props) {
     }
   };
 
+  const handleOnExportExcel = () => {
+    var wb = XLSX.utils.book_new(),
```

Comment:
we want to never use var for variable declaration. 'const' if the variable value doesnt change, 'let' otherwise